packages/trees/frontend/sirius-components-trees/src/views/TreeFiltersMenu.tsx

```diff
@@ -71,6 +71,7 @@ export const TreeFiltersMenu = ({ filters, onTreeFilterMenuItemClick }: TreeFilt
         size="small"
         ref={anchorRef}
         aria-haspopup="true"
+        title="Filters menu"
```

Comment:
Maybe just "Filters"?
For example the tooltip for `NewDocumentModalContribution` does not say "New model dialog", just "New model".

Reply:
Done